client/components/Card/Card.jsx

```diff
@@ -1,16 +1,42 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import pt from 'prop-types'
 
+import { useDispatch } from 'react-redux'
+
+import { updateCartProducts, createCart } from '../../redux/cart/actions'
+
 import './Card.scss'
 
-const Card = ({ item }) => {
-  const { name, image, price, description } = item
+const Card = ({ item, loading, cartId }) => {
+  const { name, image, price, description, _id } = item
+
+  const dispatch = useDispatch()
+
+  const handleClick = useCallback(
+    (e) => {
+      e.preventDefault()
+
+      if (loading) return
```

Comment:
@AStashevskaya Still actual about using `{` and `}` ever for single line if statement

```js
if (loading) {
  return
}